packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -75,10 +78,11 @@ usdeSalt = "usde-main"
 susdeSalt = "susde-main"
 deusdSalt = "deusd-main"
 sdeusdSalt = "sdeusd-main"
+rseliniSalt = "rselini-main"
 
 [var.missing_vars]
 salt = ""
-rusdTokenDecimals = "6"
+rusdTokenDecimals = "7"
```

Comment:
?

Reply:
hmm, where do we keep variables that track decimals for other tokens? (just hardcoded in the configs for now?)

Reply:
yes, mistake